tests/basic/BUILD

```diff
@@ -92,6 +92,35 @@ verilator_cc_library(
     vopts = [],
 )
 
+verilog_module(
+    name = "fifo",
+    srcs = ["fifo/fifo.sv"],
+    top = "fifo",
+)
+
+verilog_module(
+    name = "fifo_tb",
+    srcs = ["fifo/fifo_tb.sv"],
+    top = "fifo_tb",
+    deps = [":fifo"],
+)
+
+xsim_test(
+    name = "fifo_xsim_test",
+    module = ":fifo_tb",
+    part_number = "xczu49dr-ffvf1760-2-e",
+    verilog_flags = "--define PASSING_TEST",
+    xilinx_env = "//tests:xilinx_env.sh",
+)
+
+xsim_test(
+    name = "fifo_xsim_fail_test",
```

Comment:
Even though this is testing failure, the test should pass. Can you add an argument `expect_failure`? Default to false. Basically, the assert should fail causing the test to pass. And if the assert passes, then the test should fail.